src/cgroups/common.rs

```diff
@@ -19,7 +19,8 @@ pub const CGROUP_PROCS: &str = "cgroup.procs";
 pub const DEFAULT_CGROUP_ROOT: &str = "/sys/fs/cgroup";
 
 pub trait CgroupManager {
-    fn apply(&self, linux_resources: &LinuxResources, pid: Pid) -> Result<()>;
+    fn add_task(&self, pid: Pid) -> Result<()>;
```

Comment:
This is slightly out of the scope of this PR, However Could you provide comments on each function?

Reply:
Done